index.js

```diff
@@ -1,24 +1,79 @@
-function rot13(str) { // LBH QVQ VG!
-var shifted = 13;
-var splitted = str.split('');
-var arr = [];
-var inCharCode = splitted.forEach(function(element) {
-arr.push(element.charCodeAt(0));
-return arr;
-});
-var shifterArray = arr.map(function(element) {
-return element += shifted;
```

Comment:
Could not work properly when char code + shifted is more than char code of Z letter (resulting string should not contain not-letter characters)

Reply:
Also you should map only letters and only to other letters in your alphabet.
That's why the easiest way to implement caesar's cipher is to use custom alphabet and own indexes instead of char codes.

For char codes you could map only characters in range from A (65) to Z (90) and only to other letter in this range, i.e.
Q (81) -> A+ ((Q - A) + 13) % 26 -> 65 + ((81 - 65) + 13) % 26 -> D (68)

Reply:
If character code is less than 65 or more than 90 - just return it without shifting